zendesk/cursor.go

```diff
@@ -49,9 +49,7 @@ type response struct {
 
 func NewCursor(userName, apiToken, domain string, startTime time.Time) *Cursor {
```

Comment:
Aman mentioned [the reason](https://github.com/gopherslab/conduit-connector-zendesk/pull/2#discussion_r881910958) why we implemented this interface, despite the fact that several Go clients exist. It would be good to mention that somewhere in the docs.

Reply:
commented in the readme. done